test/engine/test_pool.py

```diff
@@ -1113,6 +1113,12 @@ def status(pool):
         lazy_gc()
         assert not pool._refs
 
+    def test_timeout_accessor(self):
+        expected_timeout = 123
+        p = self._queuepool_fixture(
+            timeout=expected_timeout)
+        assert p.timeout() == expected_timeout
```

Comment:
**mike bayer** (zzzeek) wrote:

I know this is not necessarily consistent with other tests in this module but let's make it:

   eq_(p.timeout(), expected_timeout)